publish.sh

```diff
@@ -37,7 +38,16 @@ else
     git push origin --tags
 fi
 
+# run the replacer script to make all logsites check for sampling
+git ls-files | grep '.js$' | grep -v 'bin/' | grep -v test | grep -v replacer.js | xargs node replacer.js
+
+# need a temp commit because git archive uses HEAD
+git commit -a -m 'temp commit'
```

Comment:
it'll fail bc DebugLotron everywhere (including in TChannel's test cluster or whatever) which doesn't have the right interface